src/controller/picture.controller.ts

```diff
@@ -0,0 +1,200 @@
+import {NextFunction, Response} from "express";
+import {CreatePicturesParams, CustomRequest, DeletePicturesBody} from "../interfaces/RequestInterfaces";
+import {OrderModel} from "../models/order.model";
+import ErrnoException = NodeJS.ErrnoException;
+import {PictureModel} from "../models/picture.model";
+import {OrderPictureModel} from "../models/orderPicture.model";
+import {CityModel} from "../models/city.model";
+
+const fs = require('fs');
+require("dotenv").config({path: `.env.${process.env.NODE_ENV}`})
+const ApiError = require('../exeptions/api-error')
+const cloudinary = require("cloudinary").v2
+const uuid = require('uuid')
+const path = require("path")
+const {Picture, OrderPicture, Order} = require('../models/index');
+
+interface MyFile extends File {
+    data: Buffer,
+}
+
+interface UploadCloudinaryResult {
+    asset_id: string,
+    public_id: string,
+    version: number,
+    version_id: string,
+    signature: string,
+    width: number,
+    height: number,
+    format: string,
+    resource_type: string,
+    created_at: string,
+    tags: string[],
+    bytes: number,
+    type: string,
+    etag: string,
+    placeholder: boolean,
+    url: string,
+    secure_url: string,
+    original_filename: string,
+    api_key: string
+}
+
+interface DataValues extends PictureModel {
+    url: string
+}
+
+interface Picture {
+    dataValues: DataValues
+}
+
+interface OrderPictureModelWithPicture extends OrderPictureModel {
+    picture: Picture
+}
+
+class PictureController {
+    static createOnePicture(file: MyFile, next: NextFunction) {
+        const name: string = file.name
+        const fileExtension: string | undefined = name.split('.').pop()
+        const arr: string[] = ['jpeg', 'JPEG', 'jpg', 'JPG', 'png', 'PNG'] //'BMP', 'bmp', 'GIF', 'gif', 'ico', 'ICO'
```

Comment:
What is it 'arr'? Make some names with sense. For example 'availableFileTypes' etc.